styles/digitalocean/_code_environments.scss

```diff
@@ -17,6 +17,13 @@ limitations under the License.
 @import "../theme";
 @import "../mixins";
 
+$code-label-class: "code-label" !default;
+
+.#{$code-label-class}:has(+ .code-toolbar > pre[class*="environment-"]) {
+    background: $gray9;
+    color: $gray4;
+}
```

Comment:
It looks like there is an edge-case where this isn't working, when there is no toolbar:

![image](https://github.com/digitalocean/do-markdownit/assets/12371363/87caa495-8163-477b-9ee6-7bb26ac0e16b)